Infrastructure/Helper/DbSchemaCheck.cs

```diff
@@ -73,5 +74,40 @@ private static void EnsurePaidCashColumnExistsInSalesTable(SQLiteConnection conn
                 }
             }
         }
+
+        private static void EnsureStorageAddressColumnDeletedFromAvailabilityTable(SQLiteConnection connection)
+        {
+            using (var transaction = connection.BeginTransaction())
```

Comment:
Do you really need a transaction here?

Reply:
yes, because there are used two commands 
![image](https://github.com/user-attachments/assets/4464264a-de86-46ba-a422-f2ffa10e34ae)